diff --git a/node_modules/@angular-devkit/build-angular/src/builders/app-shell/index.js b/node_modules/@angular-devkit/build-angular/src/builders/app-shell/index.js
index c2975fa..c530d00 100644
--- a/node_modules/@angular-devkit/build-angular/src/builders/app-shell/index.js
+++ b/node_modules/@angular-devkit/build-angular/src/builders/app-shell/index.js
@@ -67,16 +67,19 @@ async function _renderUniversal(options, context, browserResult, serverResult, s
         const browserIndexOutputPath = path.join(outputPath, 'index.html');
         const indexHtml = await fs.promises.readFile(browserIndexOutputPath, 'utf8');
         const serverBundlePath = await _getServerModuleBundlePath(options, context, serverResult, localeDirectory);
-        const { AppServerModule, renderModule } = await Promise.resolve().then(() => __importStar(require(serverBundlePath)));
+        const { AppServerModule, renderModule, APP_BASE_HREF } = await Promise.resolve().then(() => __importStar(require(serverBundlePath)));
         const renderModuleFn = renderModule;
         if (!(renderModuleFn && AppServerModule)) {
             throw new Error(`renderModule method and/or AppServerModule were not exported from: ${serverBundlePath}.`);
         }
+
         // Load platform server module renderer
         const renderOpts = {
             document: indexHtml,
             url: options.route,
+            extraProviders: [{provide: APP_BASE_HREF, useValue: '/my/app'}]
         };
+
         let html = await renderModuleFn(AppServerModule, renderOpts);
         // Overwrite the client index file.
         const outputIndexPath = options.outputIndexPath
